docs/source/build-run-qualcomm-ai-engine-direct-backend.md

```diff
@@ -107,22 +109,38 @@ export PYTHONPATH=$EXECUTORCH_ROOT/..
 
 ## Build
 
-An example script for below building instructions is [here](https://github.com/pytorch/executorch/blob/main/backends/qualcomm/scripts/build.sh).
+An example script for the below building instructions is [here](https://github.com/pytorch/executorch/blob/main/backends/qualcomm/scripts/build.sh).
+We recommend to use the script because the ExecuTorch build-command can change from time to time.
+The above script is actively used. It is updated more frquently than this tutorial.
+An example usage is
+```bash
+cd $EXECUTORCH_ROOT
+./backends/qualcomm/scripts/build.sh
+# or
+./backends/qualcomm/scripts/build.sh --release
+```
 
 ### AOT (Ahead-of-time) components:
 
 Python APIs on x64 are required to compile models to Qualcomm AI Engine Direct binary.
 
 ```bash
 cd $EXECUTORCH_ROOT
-# Workaround for fbs files in exir/_serialize
-cp schema/program.fbs exir/_serialize/program.fbs
-cp schema/scalar_type.fbs exir/_serialize/scalar_type.fbs
```

Comment:
Are we good with this now?

Reply:
No. But this action is more related to AOT compilation, so I moved there:
https://github.com/pytorch/executorch/pull/4590/files#diff-8bac58bd3ddef654a20549948dfea5d27f76a8ee29544328f65819cc0ccd7bcfR205